src/neuroconv/datainterfaces/behavior/deeplabcut/deeplabcutdatainterface.py

```diff
@@ -123,13 +123,13 @@ def run_conversion(
         metadata: dict
             metadata info for constructing the nwb file (optional).
         overwrite: bool, optional
-            Whether or not to overwrite the NWBFile if one exists at the nwbfile_path.
+            Whether to overwrite the NWBFile if one exists at the nwbfile_path.
         """
-        base_metadata = self.get_metadata()
-        metadata = dict_deep_update(base_metadata, metadata)
+        _metadata = self.get_metadata()
+        _metadata.deep_update(metadata)
```

Comment:
I would highly recommend reverting all these integrations within the interfaces - that can be a next step, but currently the `dict_deep_update` does have occasionally very different behavior from `deep_update` on a `DeepDict`, particularly with respect to `list` type values

Reply:
ok, but they were failing. I suppose a simpler solution would be to have `dict_deep_update` cast any DeepDicts to dicts.

Reply:
Yeah, I wouldn't be surprised if there aren't any explicit tests out there for append vs. extend usages of the previous deep update; will need to carefully examine where the assumptions and uses of that pop up in practice

Or, better solution, just starting requiring everything in NeuroConv metadata to be a dict instead of list and then we never need to worry about it